java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/RowProjectPlan.scala

```diff
@@ -115,11 +115,14 @@ object RowProjectPlan {
         val buffer = projectConfig.moduleNoneBroadcast
 
         if (hybridseJsdkLibraryPath.equals("")) {
+          JitManager.initCore()
           JitManager.initJitModule(tag, buffer.getBuffer)
```

Comment:
Why put the jit init in RowProjectPlan.scala? It'd better to move to a higher module?

I saw in other *Plan.scala, there are the same init steps for jit.

Reply:
You're right. We should add this in other planner or add the method in `initJitModule()`.